six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -52,14 +52,14 @@ six::sicd::ComplexData * asComplexData(six::Data* data)
   }
 }
 
-void writeNITF(std::string fileName, std::vector<std::string> schemaPaths,
-        six::sicd::ComplexData* data, long long imageAdr);
+void writeNITF(const std::string& pathName, const std::vector<std::string>&
+        schemaPaths, six::sicd::ComplexData* data, long long imageAdr);
 
-void writeNITF(std::string fileName, std::vector<std::string> schemaPaths,
-        six::sicd::ComplexData* data, long long imageAdr)
+void writeNITF(const std::string& pathName, const std::vector<std::string>&
+        schemaPaths, six::sicd::ComplexData* data, long long imageAdr)
```

Comment:
The latest issues are fixed.

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Thursday, January 28, 2016 4:55 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>
Subject: Re: [six-library] Fix issues from code review (#46)


In six/modules/python/six.sicd/source/six_sicd.i<http://cp.mcafee.com/d/5fHCNEe6zqb3X5QnArI6XCSm6hMUQsLLCTNPb3a9EVd79J6XRTD3hPsQsL6QkmkkTPhOUM-r4qMSpfosH-ME2BkdrFOVJl3mWsKr7c-ZKKfZvArILfL3A-LsKDtW_9Tsuod7ad-EyCJtdmVEVsVkffGhBrwqrhdFCXYyMCY-ehojd79KVIDeqR4IMzapdIxO-g8Wh-IEjOAwYKv8x7oBImrffNxM349kITiAGM5oLEECzBAQsI9CO5mUm-w0nC7C4miXb2eBN6IdCjS7a_Ia7aEqTjBMlgy9YhBcCSgVsS_tBAS2_id41FrF8fbDO8hS9oQKCy0nA7Cy2lad_gQgeNGGq8blgYQghYPbP1Ewd506ce6yAS-yraYG7>:

>

> -void writeNITF(std::string fileName, std::vector<std::string> schemaPaths,

> -        six::sicd::ComplexData* data, long long imageAdr)

> +void writeNITF(const std::string& pathName, const std::vector<std::string>&

> +        schemaPaths, six::sicd::ComplexData* data, long long imageAdr)

Pass data in by const ref rather than non-const pointer

—
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/avndxMscygArhovoKyYztwTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPoVDTJRN_HYztBVZUsDRXBQXLnVeXzP1EVhLR4kRHFGTd7bDaxVZicHs3jqpJcTvAm4TDNOb2pEVdTdAVPmEBC4pj9JAenO17ifRB2ukA7BPV48X4JyPpV-ceoX8_W0p5Z54QsICzBxcSgGT2TQ02YMYMyOnpohQK8RxIOuMVnZxgVl3mWsK2G4hfycFASO7bCTXIICMnWhEwdbt91Vs-h2eNb6BQQg2YwYQgiFhLW6y1Sdljh1qG7Cy2fCpuod41EE0NxMQkCTQjvJjV>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.